.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -63,11 +63,12 @@ Try it out yourself:
 ![image](https://user-images.githubusercontent.com/69332964/98034882-ad787100-1de5-11eb-83fd-9cb73f78beae.png)
 
 2. **Setting Parameters:** Click on "Params" and enter `color` into Key and the color you want (eg. blue) into Value. Enter `text` into the next Key row and a number (eg. 50) into Value.
+> Note on parameters: Colors are pretty hit or miss; since the Cat API is on the web, it generally adheres to HTML color names. Use [this link](https://www.cknow.com/cms/articles/what-are-the-official-html-color-names-and-hex-codes-3.html) in order to use a wider range colors for your picture! Additionally, the API can take very large words as input for the text, however only **34** characters can be seen on the picture at one time    
```

Comment:
I would first format it like this:
> **Note on parameters:** 
> * Colors are pretty hit or miss; since the Cat API is on the web, it generally adheres to HTML color names. Use [this link](https://www.cknow.com/cms/articles/what-are-the-official-html-color-names-and-hex-codes-3.html) in order to use a wider range colors for your picture! 
> * The API can take very large words as input for the text, however only **34** characters can be seen on the picture at one time .

Also, the text parameter actually refers to the *font size* and not the number of characters. See the picture here:
![image](https://user-images.githubusercontent.com/69332964/114480204-1dc6c200-9bd0-11eb-8efe-73c708eb179e.png)
Instead of providing a link, maybe just say that it only accepts "blue, red, black..." and has a max text size of 50 (I don't know what it is)

Overall, great work, but you might have to do some testing with the API to figure out the bounds. (The documentation for this is pretty bad)

Reply:
got it! thank you